### PR TITLE
#12806299 | Admin can edit their information

### PR DESCRIPTION
**Pull request by adetola-ralph**
Screenshot 2020-03-02 at 15 22 22
Screenshot 2020-03-02 at 15 22 29
Screenshot 2020-03-02 at 15 22 35
Screenshot 2020-03-02 at 15 22 46
Screenshot 2020-03-02 at 15 22 50
Screenshot 2020-03-02 at 15 22 53

### CODE REVIEW

File: src/components/Modals/EditProfileModal.vue [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+              <label for="profile-image" class="w-40 h-32 block cursor-pointer">
+                <input type="file" name="profile image" id="profile-image" class="hidden" accept="image/*" @change="onFileUpload($event);">
+                <div class="w-full h-full profile-image flex items-center justify-center" v-if="!uploadedImageSrc">
+                  <span class="text-white text-sm font-circular">Upload picture</span>
+                </div>
+                <!-- thinking of using canvas here so I can make modifications to the image after upload -->
+                <div v-bind:style="{ 'background-image': 'url('+ uploadedImageSrc +')'  }" class="w-full h-full bg-cover bg-no-repeat"></div>
+              </label>
+            </div>
+            <div class="w-2/3">
```
Comment (Saint-lawal) [inline src/components/Modals/EditProfileModal.vue]: How come there's nothing in this tag?

File: src/components/Modals/EditProfileModal.vue [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+                  <input
+                    class="w-full pl-1 py-2 field border-b border-gray-400"
+                    :class="errors.length > 0 ? 'border-red-600' : ''"
+                    type="password"
+                    id="admin-password"
+                    name="admin-password" />
+                </ValidationProvider>
+              </div>
+            </div> -->
+            <!-- <div class="flex mb-3">
```
Comment (Saint-lawal) [inline src/components/Modals/EditProfileModal.vue]: Any particular reason why this is all commented out?

File: src/components/Modals/EditProfileModal.vue (hunk 1 above)
Comment (adetola-ralph) [reply src/components/Modals/EditProfileModal.vue]: I removed the input search that was there and I am using that div as a structural element.

Comment (Saint-lawal) [reply src/components/Modals/EditProfileModal.vue]: Alright 👍

File: src/components/Modals/EditProfileModal.vue (hunk 2 above)
Comment (adetola-ralph) [reply src/components/Modals/EditProfileModal.vue]: Don't want to remove it incase we need to enable editing phone numbers.

**Pull request merged**